### ISSUE TITLE
The configure script should look for CoqEAL

### ISSUE THREAD

**Issue by erikmd** [validsdp/validsdp#3]
- The configure.ac file should be adapted to check if CoqEAL's theories are available

**Comment by proux01** [comment]
Actually, the configure script was not looking for CoqEAL because it is expected
to be buit by `make external` after configure time.

**Comment by erikmd** [comment]
@proux01 Good catch! :)
As you suggested in your last email, we'll indeed be able to remove the submodules soon (which were anyway much less practical than subtrees according to a recent experiment I performed, and deletion also doesn't seem trivial according to this [SO answer](https://stackoverflow.com/questions/1260748/how-do-i-remove-a-submodule)).
However I'd vote to wait a little bit (that all of our Coq dependencies have been released on OPAM) to remove the submodules. Because, asking users to do "git clone..." is more or less what should be automated by the submodules infrastructure that is already set-up...

Question: as this issue and my commit f9a4378f900a51f7e609d4347bf4452488cf4ca3 were actually off-topic due to the presence of `make external`, I propose to revert my commit in master if you are OK with this. (If yes, I could of course reapply it once the submodules will be removed.)
